src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const instructionalPart2Description = (
+  <div>
+    <h3>Fill in employer details</h3>
```

Comment:
other pages are using `<h4>`, might have to change other pages if `<h3>` is what we are supposed to be using